docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -163,12 +170,41 @@ The lock screen manager is configured to dismiss the lock screen when it is safe
 ##### Objective-C
```

Comment:
Making the lock screen always on is missing from Android.

Reply:
isnt that what the /toDO is for? or should i add it myself?

Reply:
ios tag was out of place fixed!